### PR TITLE
Added redmineRole attribute

### PR DESCRIPTION
**Pull request by hrasnam**

### CODE REVIEW

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import cz.metacentrum.perun.core.api.Resource;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleImplApi;
+
+/**
+ * Module for redmine role
+ * Sets the role to either Manager, Reporter or Developer
```
Comment (licehammer) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java]: It doesn't "set" the role. It only checks if the role is set to valid value.

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * @date 04.03.16.
+ */
+public class urn_perun_resource_attribute_def_def_redmineRole extends ResourceAttributesModuleAbstract implements ResourceAttributesModuleImplApi {
+    public void checkAttributeValue(PerunSessionImpl perunSession, Resource resource, Attribute attribute) throws InternalErrorException, WrongAttributeValueException {
+        String role = (String)attribute.getValue();
+
+        if (role == null) {
+            throw new WrongAttributeValueException(attribute, resource, "Attribute value is null.");
+        }
+        else if ((!role.equals("Manager")  || !role.equals("Reporter") || !role.equals("Developer")) {
```
Comment (licehammer) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java]: This condition will be always true.

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            throw new WrongAttributeValueException(attribute, resource, "Attribute value is invalid.");
```
Comment (licehammer) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java]: The exception message will be shown to end users. Please add information what the valid values are.

**Comment by licehammer** [comment]
Please use tab for indentation.

**Comment by licehammer** [comment]
Please, squash commits into a single one.

**Comment by hrasnam** [comment]
Tento commit som spravil omylom:
"Merge branch 'master' of https://github.com/hrasnam/perun"

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+ * Checks if the role has correct value of either Manager, Reporter or Developer
+ * @author Marek Hrasna marekhrasna@outlook.com
+ * @date 04.03.16.
+ */
+public class urn_perun_resource_attribute_def_def_redmineRole extends ResourceAttributesModuleAbstract implements ResourceAttributesModuleImplApi {
+	public void checkAttributeValue(PerunSessionImpl perunSession, Resource resource, Attribute attribute) throws InternalErrorException, WrongAttributeValueException {
+		String role = (String)attribute.getValue();
+
+		if (role == null) {
+			throw new WrongAttributeValueException(attribute, resource, "Attribute value is null.");
```
Comment (licehammer) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java]: Please change error message to be more comprehensible. 
You can use the same message as it's used few lines later: "Attribute value is invalid.The role can be either Manager, Reporter or Developer"

File: perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+		}
+		else if (!role.equals("Manager")  && !role.equals("Reporter") && !role.equals("Developer")) {
```
Comment (licehammer) [inline perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineRole.java]: Double space between '("Manager")' and '&&'. 
It's not that important, but when you have to change this pull request, you can fix this too.

**Comment by balcirakpeter** [comment]
problem solving in #1259

**Pull request closed without merging**